app/components/banner_component.rb

```diff
@@ -1,4 +1,8 @@
 class BannerComponent < GovukComponent::Base
+  def initialize(classes: [], html_attributes: {})
+    super(classes: classes, html_attributes: html_attributes)
+  end
```

Comment:
What's the purpose of this method? It looks the same as super?

Reply:
have a look at how other GOVUK Components have been implemented, like this one:

https://github.com/DFE-Digital/govuk-components/blob/master/app/components/govuk_component/back_link_component.rb

classes and html_attributes must always be passed in in the base class, but are optional in the subclasses